jemoji.gemspec

```diff
@@ -12,7 +12,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'jekyll', '>= 3.0'
   s.add_dependency 'html-pipeline', '~> 2.2'
   s.add_dependency 'activesupport', '~> 4.0'
-  s.add_dependency 'gemoji', '~> 2.0'
+  s.add_dependency 'gemoji', '~> 3.0'
 
```

Comment:
Oooh, good point. It might create some confusion for some users, though, as then they'd have to manually upgrade gemoji themselves. But I'm down to do that if you think that's wiser. I'd be interested in @benbalter's take.